kclvm/sema/src/resolver/global.rs

```diff
@@ -9,7 +9,6 @@ use crate::ty::{
 };
 use indexmap::IndexMap;
 use kclvm_ast::ast;
-use kclvm_ast::walker::MutSelfTypedResultWalker;
```

Comment:
![image](https://github.com/kcl-lang/kcl/assets/22744597/c7e47fe5-6a2c-4d06-8fe1-7ef90f202d47)
Linux CI failed.

Reply:
rerun and passed